app/uk/gov/hmrc/hmrcemailrenderer/templates/tcs/newTcsRenewalConfirmationMessageAlert.scala.html

```diff
@@ -1,24 +1,106 @@
-@(params: Map[String, Any])@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params,"Your tax credits renewal has been received", true, footerMessage = None, true) {
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Your tax credits renewal has been received", true, footerMessage = None, true) {
 
 <p style="font-size: 24px;line-height: 1.315789474; margin: 0 0 30px 0;">
-    Your online reference number is
+    Your reference number is
     <span style="white-space: nowrap;">@{params("renewalReference")}</span>
 </p>
 
-<h2>You do not need to do anything else</h2>
+<h2>What you can do next</h2>
+@if((params("isMultiple") == "true") && (params("anyEstimates") != "0") && (params("anyEstimates") != "1")){
```

Comment:
I'll have a look at fragments and as for combining the two the TemplateParams map takes a 
String -> String so it was implemented this way rather than using' > 1' but I'll have a look thanks.